core-services/release-controller/_releases/priv/release-ocp-4.12.json

```diff
@@ -125,25 +125,25 @@
                 "name": "release-openshift-ocp-installer-e2e-aws-upi-4.12-priv"
             }
         },
-        "azure": {
+        "azure-csi": {
             "disabled": true,
             "optional": true,
             "prowJob": {
-                "name": "periodic-ci-openshift-release-master-nightly-4.12-e2e-azure-priv"
+                "name": "periodic-ci-openshift-release-master-nightly-4.12-e2e-azure-csi-priv"
             }
         },
-        "azure-csi": {
+        "azure-ovn": {
             "disabled": true,
             "optional": true,
             "prowJob": {
-                "name": "periodic-ci-openshift-release-master-nightly-4.12-e2e-azure-csi-priv"
+                "name": "periodic-ci-openshift-release-master-ci-4.12-e2e-azure-ovn-priv"
```

Comment:
why do we have a `...-ci-...` job in this nightly blocker/informer set?

Reply:
not sure. let me see what's up?

Reply:
this is leftover from a mistake in this [commit](https://github.com/openshift/release/commit/087ae86ff0d5fe38c99af222182ab890d5a61b59).
I'll address it with it's own PR.

Reply:
it was that way before, so :shrug: no need to hold this PR.

/approve